packages/core/src/config.ts

```diff
@@ -18,6 +18,7 @@ export let config = {
   "refereeAddress": "0xfD41041180571C5D371BEA3D9550E55653671198",
   "refereeDeployedBlockNumber": 157193676,
   "refereeImplementationAddress": "0x29a7b907fdf4a9235f46d891b7aa1e7d3d35a3b6",
+  "refereeCalculationsAddress": "", // TODO Add RefereeCalculations address
```

Comment:
Is this breaking the build ?

Reply:
If you mean core build, no. When I run pnpm cli it builds/runs.